src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/RfidTags/RfidTagForm.razor

```diff
@@ -0,0 +1,50 @@
+﻿@using NTS.Judge.Blazor.Ports
+@using NTS.Domain.Setup.Entities
+
+<MudText Class="mt-2" Typo="Typo.body2">Scan Rfid Chip (optional)</MudText>
+
+<MudContainer Class="d-flex flex-row">
```

Comment:
I want to stress that you should use Mud components for layout instead of display: flex directly. `Class` properties should be used mostly for padding and margin.

Reply:
This is too much stress 😄 Especially since I couldn't find a Mud Component that sets the flex direction or flex for that matter. Any suggestions?

Reply:
See [MudStack](https://mudblazor.com/components/stack#basic-usage)